docs/source/substrafl_doc/substrafl_overview.rst

```diff
@@ -22,7 +22,7 @@ Installation
 
 .. _installation:
 
-Substrafl and Substra are compatible with Python versions 3.9, 3.10 and 3.11 on Windows, MacOS and Linux.
+Substrafl and Substra are compatible with Python versions 3.9, 3.10, 3.11 and 3.12 on Windows, MacOS and Linux.
```

Comment:
Do we still say that we are compatible with Python 3.9 if we are removing testing on it?

Reply:
We are still testing in Python 3.9 in the CI and in all other repos.